DrivewayDentDeletion/Operators/cicd-test-apic/cicd-pipeline.yaml

```diff
@@ -0,0 +1,471 @@
+    - name: build-ace-int-server-ace-chris
+      runAfter:
+        - clone-git-source
+      taskRef:
+        name: build-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-chris"
+        - name: dockerfile
+          value: "ACE-Chris.Dockerfile"
+        - name: contextPath
+          value: "Bar_files/ace-chris"
+        - name: pvc
+          value: "buildah-ace-chris"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+
+    # deploy all ace in dev namespace
+    - name: deploy-wait-ace-api
+      runAfter:
+        - build-ace-int-server-ace-api
+      taskRef:
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-api"
+        - name: releaseName
+          value: "ace-api-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+    - name: deploy-wait-ace-bernie
+      runAfter:
+        - build-ace-int-server-ace-bernie
+      taskRef:
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-bernie"
+        - name: releaseName
+          value: "ace-bernie-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+    - name: deploy-wait-ace-acme
+      runAfter:
+        - build-ace-int-server-ace-acme
+      taskRef:
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-acme"
+        - name: releaseName
+          value: "ace-acme-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+    - name: deploy-wait-ace-chris
+      runAfter:
+        - build-ace-int-server-ace-chris
+      taskRef:
+        name: deploy-task
+      params:
+        - name: imageTag
+          value: "$(params.imageTag)"
+        - name: imageName
+          value: "ace-chris"
+        - name: releaseName
+          value: "ace-chris-int-srv"
+        - name: releaseScript
+          value: release-ace-integration-server.sh
+        - name: namespace
+          value: "{{NAMESPACE}}"
+      workspaces:
+        - name: git-source
+          workspace: git-source
+
+    # test end-to-end API in dev namespace
+    - name: test-e2e-api-in-dev-namespace
+      runAfter:
+        - deploy-wait-ace-chris
```

Comment:
I think this should run after `apic-resource-config-dev`

Reply:
yes..